e2e/sqlserver/helmfile.yaml

```
@@ -0,0 +1,100 @@
+    hooks:
+      - events: ["presync"]
+        showlogs: true
+        command: kubectl
+        args:
+          - create
+          - namespace
+          - '{{ requiredEnv "E2E_NS" }}'
+      - events: ["postuninstall"]
+        showlogs: true
+        command: kubectl
+        args:
+          - delete
+          - namespace
+          - '{{ requiredEnv "E2E_NS" }}'
+    values:
+      - nameOverride: '{{ requiredEnv "E2E_CONTEXT_ID" }}'
+      - sa_password: MyPassword1!
+
+  - name: cerbos
+    namespace: '{{ requiredEnv "E2E_NS" }}'
+    needs: ["sqlserver"]
+    createNamespace: true
+    labels:
+      e2e-run: '{{ requiredEnv "E2E_RUN_ID" }}'
+      e2e-ctx: '{{ requiredEnv "E2E_CONTEXT_ID" }}'
+    chart: '{{ requiredEnv "E2E_SRC_ROOT" }}/deploy/charts/cerbos'
+    hooks:
+      - events: ["presync"]
+        showlogs: true
+        command: kubectl
+        args:
+          - create
+          - secret
+          - tls
+          - 'cerbos-certs-{{ requiredEnv "E2E_CONTEXT_ID" }}'
+          - '--cert={{ requiredEnv "E2E_SRC_ROOT" }}/internal/test/testdata/server/tls.crt'
+          - '--key={{ requiredEnv "E2E_SRC_ROOT" }}/internal/test/testdata/server/tls.key'
+          - '--namespace={{ requiredEnv "E2E_NS" }}'
+      - events: ["postuninstall"]
+        showlogs: true
+        command: kubectl
+        args:
+          - delete
+          - secret
+          - 'cerbos-certs-{{ requiredEnv "E2E_CONTEXT_ID" }}'
+          - '--namespace={{ requiredEnv "E2E_NS" }}'
+    values:
+      - nameOverride: '{{ requiredEnv "E2E_CONTEXT_ID" }}'
+      - image:
+          repository: '{{ env "E2E_CERBOS_IMG_REPO" | default "ghcr.io/cerbos/cerbos" }}'
+          tag: '{{ env "E2E_CERBOS_IMG_TAG" | default "dev" }}'
+      - volumes:
+        - name: cerbos-auditlog
+          emptyDir: {}
+      - volumeMounts:
+        - name: cerbos-auditlog
+          mountPath: /audit
+      - cerbos:
+          tlsSecretName: 'cerbos-certs-{{ requiredEnv "E2E_CONTEXT_ID" }}'
+          logLevel: DEBUG
+          config:
+            server:
+              playgroundEnabled: true
+              adminAPI:
+                enabled: true
+                adminCredentials: 
+                  username: cerbos
+                  passwordHash: JDJ5JDEwJC5BYjQyY2RJNG5QR2NWMmJPdnNtQU93c09RYVA0eFFGdHBrbmFEeXh1NnlIVTE1cHJNY05PCgo=
+            auxData:
+              jwt:
+                disableVerification: true
+            schema:
+              enforcement: reject
+            audit:
+              enabled: true
+              accessLogsEnabled: true
+              decisionLogsEnabled: true
+              backend: local
+              local: 
+                storagePath: /audit/cerbos
+            storage:
+              driver: "sqlserver"
+              sqlserver:
+                url: 'sqlserver://cerbos_user:ChangeMe(1!!)@mssqllatest-{{ requiredEnv "E2E_CONTEXT_ID" }}.{{ requiredEnv "E2E_NS" }}.svc.cluster.local:1433?database=cerbos'
```

This would be just `<context_id>.<namespace>.svc.cluster.local` because you set the `nameOverride` value to `E2E_CONTEXT_ID` and that changes the name of the service to just `<context_id>`: https://github.com/cerbos/cerbos/blob/e3bea416d6f77738c48c48d9c8d039b15915c404/e2e/sqlserver/chart/templates/_helpers.tpl#L17

```sh
k get svc -n e2e-ubtll
NAME               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
cerbos-sqlserver   ClusterIP   10.96.155.142   <none>        3592/TCP,3593/TCP   80m
sqlserver          ClusterIP   10.96.248.248   <none>        1433/TCP            82m
```


```suggestion
                url: 'sqlserver://cerbos_user:ChangeMe(1!!)@{{ requiredEnv "E2E_CONTEXT_ID" }}.{{ requiredEnv "E2E_NS" }}.svc.cluster.local:1433?database=cerbos'
```